package.json

```diff
@@ -340,11 +341,20 @@
 		"terser-webpack-plugin": "^1.2.3",
 		"ts-jest": "^29.1.0",
 		"ts-loader": "^8.0.11",
+		"ts-morph": "^21.0.1",
 		"ts-node": "^10.9.1",
 		"typescript": "^5.2.2",
 		"webpack": "^4.41.5",
 		"webpack-bundle-analyzer": "^3.1.0",
 		"webpack-cli": "^3.3.1",
 		"webpack-manifest-plugin": "^2.0.4"
+	},
+	"overrides": {
+		"@ts-rest/express": {
+			"@ts-rest/core": "3.30.52"
+		},
+		"@ts-rest/open-api": {
+			"@ts-rest/core": "3.30.52"
+		}
```

Comment:
this makes it use our forks version (see https://github.com/pubpub/ts-rest for details on the versioning).
otherwise pnpm'll complain